src/network/core/tcp_coordinator.h

```diff
@@ -263,6 +265,28 @@ class NetworkCoordinatorSocketHandler : public NetworkTCPSocketHandler {
 	 */
 	virtual bool Receive_GC_STUN_CONNECT(Packet *p);
 
+	/**
+	 * Game Coordinator informs the client of the NewGRFs in the lookup table.
+	 * This packet is sent after a CLIENT_LISTING reqest, but before GC_LISTING.
+	 *
+	 *  uint32   Listing version.
```

Comment:
I have been thinking how to name this field. And I think `cursor` might be a wise name. So `Lookup table cursor`?